setup.py

```diff
@@ -1,5 +1,6 @@
 import os
 from setuptools import setup, find_namespace_packages
+from Cython.Build import cythonize
```

Comment:
Since our package also targets different platforms such as Mac and Windows. We should think about the necessary and the difficulty to using Cython.

Reply:
Hi, I think you are looking at an old commit.  This isn't actually in the pull request.  I did intentionally leave the commit in the commit history so it is available if we want to implement in future.